.github/workflows/ci.yml

```diff
@@ -54,6 +54,8 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
+      - name: Print locked memory limits
+        run: ulimit -l; ulimit -H -l
```

Comment:
Can this change be explained? Why are soft and hard limits printed?

Reply:
I was troubleshooting some tests that were failing in CI, where both limits turn out to be set to 64 k (that's only 16 pages). I printed the hard limit on the off chance that we could raise the soft limit to something bigger than it's set to in the runner shell.

This limit gives you the total size of memory pages a process is allowed to lock, which the memory referenced in `register_buffers` is counted to. I've adjusted the tests to not require more than a fraction of that limit, and to not use very small buffers that may be laid out on many more pages than their total size suggests due to fragmentation. The printouts can be removed now.

Reply:
Removed in https://github.com/tokio-rs/tokio-uring/pull/190/commits/91ce0dcbbf15892cac4dd903e7e3d0009f5c8fa4